custom-eureka-client/README.md

```diff
@@ -0,0 +1,56 @@
+# Access Azure Spring Cloud managed Config Server

```

Comment:
folder name doesn't match content?

Reply:
also add an entry in the repo's README to link to this sub-project?

Reply:
updated.